lib/jsdom/living/nodes/HTMLScriptElement-impl.js

```diff
@@ -11,13 +11,15 @@ const domSymbolTree = internalConstants.domSymbolTree;
 class HTMLScriptElementImpl extends HTMLElementImpl {
   constructor(args, privateData) {
     super(args, privateData);
-    this.addEventListener("DOMNodeInsertedIntoDocument", () => {
-      if (this.src) {
-        resourceLoader.load(this, this.src, this._eval);
-      } else {
-        resourceLoader.enqueue(this, this._ownerDocument.URL, this._eval)(null, this.text);
-      }
-    });
+  }
+
+  _attach() {
```

Comment:
Nice! Does this fix something in particular, or is it just general cleanup? Always happy to remove internal mutation event uses.